src/Epp.tex

```diff
@@ -7204,7 +7204,7 @@ \subsubsection{Exercise 22}
 All Java programs have at least 5 lines.
 
 \begin{proof}
-$\fa x$, if $x$ is a Java program, then $x$ has at least 5 lines.
+$\fa$ program $x$, if $x$ is written in Java, then $x$ has at least 5 lines.
```

Comment:
Makes sense, I didn't pay enough attention to the underlined placeholder in "for all ____ x".